lib/tpl/allTubes.php

```diff
@@ -20,7 +20,7 @@
                     </tr>
                 </thead>
                 <tbody>
-                    <?php foreach ($tubes as $tubeItem): ?>
+                    <?php foreach ((is_array($tubes) ? $tubes : []) as $tubeItem): ?>
```

Comment:
@drull755 Please change this line to be PHP5.2 compatible as this library is still compatible with PHP5.2